.travis.yml

```diff
@@ -21,8 +21,8 @@ matrix:
     - rust: nightly
 
 script:
-  - cargo build --features "$CARGO_FEATURES"
-  - cargo test --features "$CARGO_FEATURES"
+  - cargo build --no-default-features --features "$CARGO_FEATURES"
+  - cargo test --no-default-features --features "$CARGO_FEATURES"
```

Comment:
this would *only* test with selected features. We need a test without any features, at least

Reply:
I'm not quite sure what you mean. 

- In one row in the Travis "matrix" is with `CARGO_FEATURES=""`. And `cargo build --no-default-features --features ""` builds without any features, right? 
- In the other row of the matrix we test with `CARGO_FEATURES="mint serde rand"`, resulting in `cargo build --no-default-features --features "mint serde rand"`.

So as far as I understand it, it is tested once with `rand` and once without.

Reply:
ah right, I missed that, sorry about the confusion!